README.rst

```diff
@@ -33,7 +33,31 @@ You can also set the `TESTING` flag to `True` so no mails are sent
 Deploy it
 =========
 
-To deploy it, I'm using gunicorn and supervisord.
+With Apache and mod_wsgi
+------------------------
+
+1. Install Apache and mod_wsgi::
+
+    $ sudo apt-get install apache2
```

Comment:
Maybe the best here is to stay distro agnostic and just say 'Install Apache and activate mod_wsgi' and maybe just add a "e.g. libapache2-mod-wsgi(-py3) for Debian based and mod_wsgi for RH based distributions" because everyone knows how to install apache :)